packages/core/src/controllers/ConvertController.ts

```diff
@@ -0,0 +1,664 @@
+import { subscribeKey as subKey } from 'valtio/utils'
```

Comment:
Too big file, maybe we could separate some functions to different controllers/utils?